### PR TITLE
Query and update

### PR DESCRIPTION
**Pull request by AlaaAlsalem**
Here are the things I added to this repo
> Update schema by adding a new row 
write transaction commands 
write queries to view data from the database 
here are an attached file for [screenshots ](https://github.com/AlaaAlsalem/Vet-clinic/tree/query_and_update/Screenshots%20for%20select%20query)of the queries results

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -9,3 +9,81 @@ select name, escape_attempts from animals where weight_kg > 10.5;
...
+values('Blossom','1998-10-13','1',3,17);
+
+
+
+
+
+ insert into animals
+ (name,date_of_birth,neutered,escape_attempts,weight_kg)
+ values('Ditto','2022-05-14','1',4,22);
+
```
Comment (CollinsTatang) [inline queries.sql]: - [x] Please create a separate file for everything that populates the tables with data.
 The above query should be saved in the `data.sql` file.

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @AlaaAlsalem, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

## Required changes 
It's ok to have bugs in your code. Please, see this code review as a chance for you to learn practical coding skills and learn new tips that can improve your code quality and help you get a better job.
_Check the comments under the review._
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (AlaaAlsalem) [reply queries.sql]: Thanks for your feedback, I just solve it

**Review by Lameck1 — APPROVED** [review]
Hi, @AlaaAlsalem,

You've done well working on the concerns of the previous review 👍🏼.
The tasks for this milestone are now successfully implemented 💯.

## Highlights
- Professional README ✔️
- Clean code ✔️ 
- Good commit messages ✔️ 

All the best with the next project.

You are free to merge this branch :shipit:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**Please, remember to tag me in your question so I can receive the notification.**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**